build-system/babel-plugins/babel-plugin-transform-jss/test/fixtures/transform-assertions/should-transform/input.js

```diff
@@ -0,0 +1,22 @@
+/**
+ * Copyright 2020 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */ 
+import {createUseStyles} from 'react-jss'
+
+export const JSS = {button: {fontSize: 12}};
```

Comment:
I know I'm getting greedy here, but wanted to ask still a couple of questions. It's fine to keep as is for now if it's possible but too hard.

1. Is there a way to pick up the first argument in the call to `createUserStyles` below to get and evaluate these data? I.e. can we go without requiring `export const JSS`?
2. As a compromise, can we remove the requirement for the `JSS` constant be exported?

Reply:
Both of those are doable. For expediency sake, I'll remove the `export` since theres no reason thats necessary now.
If you prefer not even having a JSS variable, I'd be happy to change that in a followup PR

Reply:
actually, i think this may be very easy. trying it now

Reply:
Done